include/ck_tile/ops/epilogue/cshuffle_epilogue.hpp

```diff
@@ -154,6 +181,14 @@ struct CShuffleEpilogue
                                               tile_distribution_pattern::thread_raked>;
         constexpr auto dram_tile_distribution = TileEncodingPattern::Make2DStaticTileDistribution();
 
+        auto d_dram_small_window = generate_tuple(
+            [&](auto idx) { return make_tile_window(ds_dram_window[idx], dram_tile_distribution); },
```

Comment:
You have to set tile window lengths here as for lds windows. Otherwise you have in here window of size : MPerBlock x Nperblock.